src/main/java/PlopBot.java

```diff
@@ -0,0 +1,314 @@
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.temporal.TemporalAdjusters;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class PlopBot {
+    // Static variables
+    private static final String name = "plopBot";
+    private static final String HORIZONTAL_LINE = "//" + "\u2500".repeat(50);
+    private static final String ECHO_LINE = "    " + "\u2500".repeat(48);
+    private static Scanner scanner = new Scanner(System.in);
+    private static ArrayList<Task> tasks;
+    private static String userName;
+
+    public static void main(String[] args) {
+        welcomeMessage();
+        tasks = new ArrayList<>();
+        inputReader();
+    }
+
+    // Welcome message when the user runs the program.
+    static void welcomeMessage() {
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println("Hello! I'm " + name + ".");
+        System.out.println("What can I do for you today?\n");
+    }
+
+    // Farewell message when the user runs the program.
+    static void farewellMessage() {
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println("Thank you for choosing " + name + ". Have a great day!\n");
+    }
+
+    /**
+     * Reads the user's inputs and calls processInput().
+     */
+    static void inputReader() {
+        String userInput;
+
+        while (true) {
+            userInput = scanner.nextLine().trim();
+            // processInput(userInput);
+
+            if (processInput(userInput)) {
+                break;
+            }
+        }
+
+        scanner.close();
+    }
+
+    /**
+     * Processes the user's inputs.
+     * @param userInput
+     */
+    static boolean processInput(String userInput) {
+        if (userInput.toLowerCase().startsWith("mark") || userInput.toLowerCase().startsWith("unmark")) {
+            toggleMark(userInput);
+            return false;
+        }
+
+        String[] parts = userInput.split(" ", 2);
+        String command = parts[0].toLowerCase();
+        String details = parts.length > 1 ? parts[1] : "";
+
+        switch (command) {
```

Comment:
For your switch statements, you could follow the following style set by the couse in this [link](https://se-education.org/guides/conventions/java/basic.html#layout)!